edgepi-dac.html

```diff
@@ -108,39 +150,65 @@
       <option value="AOUT8">A-OUT8</option>
     </select>
   </div>
-  <div class="form-row method">
-    <label for="node-input-method">Method:</label>
-    <select id="node-input-method">
-      <option value="write">Write Voltage</option>
-      <option value="read">Read Voltage</option>
-    </select>
+  <div class="form-row">
+    <input
+      type="radio"
+      name="gainOption"
+      id="node-input-gain1"
+      value="gain1"
+      checked
+    />
+    <label for="node-input-gain1">Gain 1</label><br />
   </div>
-  <div class="form-row voltage">
-    <label for="node-input-voltage">Voltage to write: </label>
-    <input type="number" id="node-input-voltage" placeholder="0-10" min="0" max="10" step="0.1"required>
-
+  <div class="form-row">
+    <input type="radio" name="gainOption" id="node-input-gain2" value="gain2" />
+    <label for="node-input-gain2">Gain 2</label><br />
+  </div>
+  <div class="form-row voltageGain2">
+    <label for="node-input-voltageGain2">Output voltage: </label>
+    <input
+      type="number"
+      id="node-input-voltageGain2"
+      placeholder="0-10"
+      min="0"
+      max="10"
+      step="0.1"
+    />
+  </div>
+  <div class="form-row voltageGain1">
+    <label for="node-input-voltageGain1">Output voltage: </label>
+    <input
+      type="number"
+      id="node-input-voltageGain1"
+      placeholder="0-5"
+      min="0"
+      max="5"
+      step="0.1"
+    />
   </div>
 </script>
 
-
 <script type="text/html" data-help-name="dac">
-	<p>Reads or writes a digital voltage to a given output channel on the EdgePi.</p>
-	<h3>Properties</h3>
-	<dl class="message-properties">
-		<dt> RPC Server</dt>
-		<dd>The connection to your EdgePi's RPC Server. </dd>
+  <p>
+    Generate analog output voltage based on voltage provided.
+  </p>
+  <h3>Properties</h3>
+  <dl class="message-properties">
+    <dt>RPC Server</dt>
+    <dd>The connection to your EdgePi's RPC Server.</dd>
     <dt>Channel</dt>
-		<dd>Select channel for analogue output read/write.</dd>
-    <dt>Method</dt>
-		<dd>Select method to either read or write.</dd>
-    <dt> Voltage to write<span class="property-type">number</span></dt>
-		<dd>The voltage to write to a selected channel, bound between 0 V and 10 V</dd>
-	</dl>
-	<h3>Inputs</h3>
-		<dl class="message-properties">
-			Any message can be used to trigger this node.
-		</dl>
-	<h3>Outputs</h3>
+    <dd>Select channel for analogue output read/write.</dd>
```

Comment:
just say: `Select channel for analogue output`

Reply:
Thanks for catching that!